variables.ini.dist

```diff
@@ -59,6 +60,8 @@ GLOBUS_TOKEN_URL = ; https://globus.server.org/token
 GLOBUS_AUTH_URL = ; https://globus.server.org/authorize
 OAUTH_CLIENT_KEY = ; oauth_key
 OAUTH_CLIENT_SECRET = ; oauth_secret
+KEYSTONE_SERVER = ; "https://my-openstack.cloud.org:5000"
+KEYSTONE_DOMAIN_NAME = ; "default"
```

Comment:
Suggestion: Including explanation on when/how/why to use variables could be done in clank (or wherever we are including docs)